docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -2,16 +2,16 @@
 Submitting contributions
 ========================
 
-We're always grateful for contributions to Django's code. Indeed, bug reports
-with associated contributions will get fixed *far* more quickly than those
-without a solution.
+We are always grateful for patches to Django's code.
+To ensure a smooth process, please create a ticket on our `ticket tracker`_. before submitting your patch.
+Bug reports with associated tickets and contributions will be fixed *far* more quickly than those without.
```

Comment:
I'd avoid suggesting that bugs are occasionally fixed without tickets. Typos and minor optimizations are usually merged without one, but we don't necessarily need to mention that -- I'd keep it clear that all behavior changes, including bugfixes, need accepted tickets.